app/styles/components/speaker-list.scss

```diff
@@ -13,3 +13,8 @@
 .word-break {
   overflow-wrap: break-word;
 }
+
+.speakerImg {
+	height: 220px;
+	width: 220px;	
```

Comment:
Wasn't the UI Class handling the width ?

Reply:
`ui medium` was handling the width , and i tried following to override `medium` within class but not working.
tried like -
```
 .speakerImg {
	height: 220px;
}
<img alt="speaker" class="ui medium rounded image speakerImg" src="{{if speaker.thumbnailImageUrl speaker.thumbnailImageUrl (if speaker.photoUrl speaker.photoUrl '/images/placeholders/avatar.png')}}">

```

Reply:
```
but the same was working if i do -
#speakerImg {
	height: 220px;
}
<img class = "ui medium rounded image" id="speakerImg"  >

```
but travis ci showed that id is not allowed to use.